gammapy/estimators/parameter.py

```diff
@@ -188,7 +188,7 @@ def estimate_errn_errp(self, datasets, parameter):
             f"{parameter.name}_errn": res["errn"],
         }
 
-    def estimate_scan(self, datasets, parameter):
+    def estimate_scan(self, datasets, parameter, show_pbar=False):
```

Comment:
`with pbar` is not implemented in this script, if we needed it somewhere let's add it, if not let's remove the show_pbar

Reply:
I think it makes sense to leave it, because it is passed to `fit.stat_profile` a few lines later. right?

Reply:
yep, you're right